### PR TITLE
Hosting Panel: Stub out UI

### LINKED ISSUE #36280: Hosting: SFTP Information Card and Database Access Card UI [3]
Screen Shot 2019-09-24 at 4 34 21 PM

This is a task for the UI pieces only. Feel free to put in dummy information for prototyping.

Please use Calypso components like https://wpcalypso.wordpress.com/devdocs/design/card appropriately. We may need new icon assets.

### PR DESCRIPTION
**Pull request by jeraldjuice**
### Summary

Stubbing out the UI section for the Atomic hosting panel. Also adding some Material Icon assets.

Some things to be addressed in a future PR:
- Routing still needs to be reliant on an endpoint for conditionally allowing a user to navigate to the panel.
- Dummy info needs to be replaced with info from the API endpoint.

<hr>

### Screenshots

Screen Shot 2019-10-06 at 11 25 50 PM

<hr>

### Testing

- Spin up a local Calypso dev environment.
- Go to `/hosting`.
- Select an Atomic site. Make sure that you see the new cards.

Fixes #36280

### CODE REVIEW

File: client/my-sites/hosting/index.js [hunk 1]
```diff
@@ -7,11 +7,19 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import { siteSelection, sites } from 'my-sites/controller';
+import { navigation, siteSelection, sites } from 'my-sites/controller';
```
Comment (gwwar) [inline client/my-sites/hosting/index.js]: Ah, navigation for the sidebar! :D

File: client/my-sites/hosting/main.js [hunk 2]
```diff
@@ -5,7 +5,87 @@
...
 import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import { localize } from 'i18n-calypso';
 
-const Hosting = () => <p>Placeholder</p>;
+/**
+ * Internal dependencies
+ */
+
```
Comment (gwwar) [inline client/my-sites/hosting/main.js]: Maybe try and remove the whitespace here between the external/internal dependency comments, to the first import. It helps a bit with codemods to not deal with an extra newline

File: client/my-sites/hosting/main.js [hunk 3]
```diff
@@ -5,7 +5,87 @@
...
+	const dummyInfo = {
+		[ translate( 'URL' ) ]: 'sftp1.wordpress.com',
+		[ translate( 'Port' ) ]: 22,
+		[ translate( 'Username' ) ]: 'test',
+		[ translate( 'Password' ) ]: 'test',
+	};
+
+	return (
+		<Main className="hosting is-wide-layout">
+			<PageViewTracker path="hosting/:site" title="Hosting" />
```
Comment (gwwar) [inline client/my-sites/hosting/main.js]: We can verify analytics calls in console by calling:

```
localstorage.debug = 'calypso:analytics:*';
```
and refreshing the page.

Screen Shot 2019-10-02 at 12 39 36 PM

Do you mind double-checking if we have a site type of `Atomic` or if we just bucket everyone else as `Jetpack`? This is likely unrelated, but we might have some page stats tracking issue we could file an issue for / fix.

File: client/my-sites/hosting/main.js [hunk 4]
```diff
@@ -5,7 +5,87 @@
...
+					<div className="hosting__card-icon-col">
+						<MaterialIcon className="hosting__card-icon" icon="cloud" size={ 48 } />
+					</div>
+					<div className="hosting__card-col">
+						<CardHeading>{ translate( 'SFTP Information' ) }</CardHeading>
+						<p>
+							{ translate( "Access and edit your website's files directly using an FTP client." ) }
+						</p>
+						{ map( dummyInfo, ( data, title ) => (
+							<DataSection { ...{ data, title } } />
```
Comment (gwwar) [inline client/my-sites/hosting/main.js]: When we iterate over items, we need a unique key. https://reactjs.org/docs/lists-and-keys.html#keys

We have a static order, so index should be okay.

Screen Shot 2019-10-02 at 12 39 15 PM

File: client/my-sites/hosting/style.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+		@include breakpoint( '>1280px' ) {
+			width: calc( 50% - 10px );
+		}
+	}
+}
+
+.hosting__card-icon-col {
+    margin-right: 24px;
+    height: 100%;
+}
```
Comment (gwwar) [inline client/my-sites/hosting/style.scss]: Minor, but add a newline :D

**Comment by gwwar** [comment]
Oh and before you get too deep into API data boilerplate @jeraldjuice do you mind create a PR to add the sub-section to the sidebar?

**Comment by shaunandrews** [comment]
I think this looks fine as-is, but I wanted to suggestion some small improvements:

File: client/my-sites/hosting/main.js (hunk 3 above)
Comment (jeraldjuice) [reply client/my-sites/hosting/main.js]: Looks like we're just bucketing everything into `jetpack`, it seems. Wasn't able to get another site type to show up, then found this:

Screen Shot 2019-10-06 at 11 37 24 PM

**Comment by jeraldjuice** [comment]
How it looks after design update with latest push:

Screen Shot 2019-10-06 at 11 25 50 PM

**Pull request merged**